db/memtable.cc

```diff
@@ -842,6 +826,105 @@ bool MemTable::Get(const LookupKey& key, std::string* value, Status* s,
   return found_final_value;
 }
 
+void MemTable::GetFromTable(const LookupKey& key, std::string* value, Status* s,
```

Comment:
Put all input arguments before output arguments as per convention. I think key, max_covering_tombstone_seq and callback are the input arguments. Maybe pass max_covering_tombstone_seq by value so its clear that its an input argument.